src/modules/display.js

```diff
@@ -1,22 +1,72 @@
-const data = [{ name: 'Name', score: 100 },
-  { name: 'Name', score: 20 },
-  { name: 'Name', score: 50 },
-  { name: 'Name', score: 78 },
-  { name: 'Name', score: 125 },
-  { name: 'Name', score: 77 },
-  { name: 'Name', score: 24 }];
+const userName = document.getElementById('user-name');
+const userScore = document.getElementById('user-score');
 
-const bourd = document.querySelector('.scores');
+const player = {
+  user: String,
+  score: Number,
+};
 
-let itemElements = '';
+const gameName = {
+  name: 'Random scores',
+};
 
-export default class UI {
-    static display = () => {
-      data.map((scores) => {
-        itemElements += `<li class="item"> ${scores.name}:${scores.score} </li>`;
+export default class ApiOp {
+  static newGame = async () => {
+    const reply = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/',
+      {
+        method: 'POST',
+        body: JSON.stringify(gameName),
+        headers: {
+          'content-type': 'application/json',
+        },
+      });
+    const data = await reply.json();
+    return data;
+  }
```

Comment:
- [ ] I think these two functions **gameName** and **ApiOp.newGame** are not functional in this project, You may Kindly remove them to keep our code cleaner and short 👍 👍